assets/js/Line.test.js

```diff
@@ -414,3 +414,81 @@ test('Sign config is not affected by batch updates if sign does not support mode
     },
   });
 });
+
+test('can toggle chelsea boarding announcements on and off on Silver Line page', () => {
+  const now = Date.now();
+  const signs = {};
+
+  const currentTime = now + 2000;
+  const line = 'Silver';
+  const signConfigs = {};
+  const setConfigs = jest.fn(() => true);
+  const chelseaBridgeAnnouncements = 'off';
+  const setChelseaBridgeAnnouncements = jest.fn(() => true);
+  const setConfiguredHeadways = () => {};
+  const readOnly = false;
+  const configuredHeadways = {};
```

Comment:
Aside from `setChelseaBridgeAnnouncements`, it doesn't seem like we use any of these variables beyond the `React.createElement` that comes immediately after. Could we consolidate them? I think this might be more readable since it would take up much less vertical space.

```js
React.createElement(
  Line,
  {
    signs: {},
    currentTime: Date.now() + 2000,
    line: 'Silver',
    // ...
  }
)
```